### DISCUSSION TITLE
Breadcrumbs componenet

### DISCUSSION THREAD

**Discussion by PawelCiucias** [buefy/buefy#3149] [Q&A]
I built my own Vue.js custom breadcrumbs control; then while refactoring my codebase, I thought why am I using a custom one, when i am using buefy. I see one on your documentation page,

![buefy breadcrumbs](https://user-images.githubusercontent.com/7978812/104757202-09a02f80-575d-11eb-86d3-1ec9c61d5553.png)

but i can't seem to find it listed under UI components

**Comment by mouadTaoussi** [comment]
I Have implemented and PR it, [see](https://github.com/buefy/buefy/pull/3464)
